README.md

```diff
@@ -156,17 +160,78 @@ function Tickler() {
 ```
 
 Just as in JavaScript, when we handle events in React, we can provide an `event`
-parameter to our event handler callback function. When the button is clicked, we
-can access all the information about the event (the event `target`, mouse
-coordinates via `clientX` and `clientY`, etc).
+parameter to our event handler callback function. Now that we're using TypeScript,
+however, we also have to type our `event` so TypeScript knows what kind of event 
+the function should expect. Let's take a moment to discuss typing events before 
+we get back to the `onClick`.
+
+### Event Types
+ 
```

Comment:
This section seems pretty clear, but there is a lot to digest. A couple of suggestions:
- Point out that the event type is of the form `React.event-type<element-type>` so we need to determine the type of the event as well as the type of element the event is tied to. Then continue with the use of the IDE to find the info as shown in the gif
- End the section with a summary of the steps, perhaps in bullet form